scripts/Connector/Connector_Harwin/conn_harwin_m20-781xx45_smd_top_dual_row.py

```diff
@@ -92,13 +92,19 @@ def generate_footprint(pins, configuration):
     #
     # THT Pegs
     #
-    kicad_mod.append(PadArray(start=[-1.27, -B/2], initial="", pincount=pins,
-        y_spacing=pitch, type=Pad.TYPE_NPTH, shape=Pad.SHAPE_CIRCLE,
-        size=peg_drill_tht, drill=peg_drill_tht, layers=Pad.LAYERS_NPTH))
-
-    kicad_mod.append(PadArray(start=[1.27, -B/2], initial="", pincount=pins,
-        y_spacing=pitch, type=Pad.TYPE_NPTH, shape=Pad.SHAPE_CIRCLE,
-        size=peg_drill_tht, drill=peg_drill_tht, layers=Pad.LAYERS_NPTH))
+    # kicad_mod.append(PadArray(start=[-1.27, -B/2], initial="", pincount=pins,
+        # y_spacing=pitch, type=Pad.TYPE_NPTH, shape=Pad.SHAPE_CIRCLE,
+        # size=peg_drill_tht, drill=peg_drill_tht, layers=Pad.LAYERS_NPTH))
+
+    # kicad_mod.append(PadArray(start=[1.27, -B/2], initial="", pincount=pins,
+        # y_spacing=pitch, type=Pad.TYPE_NPTH, shape=Pad.SHAPE_CIRCLE,
+        # size=peg_drill_tht, drill=peg_drill_tht, layers=Pad.LAYERS_NPTH))
+
```

Comment:
please do not keep outdated code around in comment form. This is what the git history is there for.

In general: might it be better to make pad array more powerful instead of reimplementing this here?